adev/src/content/examples/i18n/src/app/app.component.ts

```diff
@@ -23,4 +35,7 @@ export class AppComponent {
   other() {
     this.gender = 'other';
   }
+  toggleDisplay() {
+    this.toggle.update((toggle) => !this.toggle);
```

Comment:
```suggestion
    this.toggle.update((toggle) => !this.toggle());
```

`this.toggle` is a signal. Do we have to call it to get value?

Reply:
Thanks @JeanMeche @IKatsuba